intake/catalog/tests/test_local.py

```diff
@@ -431,3 +431,24 @@ def test_no_plugins():
     s = cat.tables7
     with pytest.raises(ValueError):
         s()
+
+
+def test_getitem_and_getattr():
+    from intake.source.csv import CSVSource
+    fn = abspath('multi_plugins.yaml')
```

Comment:
Should use `os.join(dirpath(__file__), 'multi_plugins.yaml')` ?

Reply:
Sure. I blindly copied this from another test. Will update all occurrences.

Reply:
Oh actually, that's what `abspath` does. (It's locally defined convenience function.)